pkg/commands/compute/deploy.go

```diff
@@ -354,13 +360,22 @@ func (c *DeployCommand) Exec(in io.Reader, out io.Writer) (err error) {
 //
 // NOTE: It also validates if the package size exceeds limit:
 // https://docs.fastly.com/products/compute-at-edge-billing-and-resource-limits#resource-limits
-func validatePackage(data manifest.Data, pathFlag string, errLog fsterr.LogInterface) (pkgName, pkgPath string, err error) {
+func validatePackage(data manifest.Data, pathFlag string, errLog fsterr.LogInterface, out io.Writer) (pkgName, pkgPath string, err error) {
 	err = data.File.ReadError()
 	if err != nil {
-		if errors.Is(err, os.ErrNotExist) {
-			err = fsterr.ErrReadingManifest
+		if pathFlag == "" {
```

Comment:
Just to confirm here: We are going to read the fastly.toml in the current directory, but if we can't find it, we then use the fastly.toml in the tar.gz? 

The reason I asked, is because I realized using the fastly.toml in the tar.gz would be a breaking change, unless we make the default the current behavior, and used this as a fail safe 😄

Reply:
@torch2424 

So yes, the CLI will try to read the fastly.toml from the current directory (which might be an actual project directory, and so the file will exist, or it might be a parent directory as in your use case scenario). 

If the fastly.toml file **_is_** there in the current directory and we've been able to read it successfully, then `data.File.ReadError()` will be `nil` and so we will just skip over all this new code that tries to read the fastly.toml from inside the package .tar.gz archive file.

BUT it will only do that if the `--path` flag is set. Otherwise if the user doesn't specify a `--path` flag, then we'll just return the original error that occurred when we tried reading the fastly.toml from the user's current directory.

Reply:
Perfect! That's exactly what I had in mind! Thank you! Let's 🚢 😄 🎉